javascript/javascript2/week3/fastestPresser.js

```diff
@@ -68,4 +74,6 @@ newGameButton.addEventListener("click", () => {
     gameStatus.innerHTML = "";
     countOfS.innerHTML = "";
     countOfL.innerHTML = "";
+    sCanvas.remove();
+    lCanvas.remove();
```

Comment:
By removing these, you can't show confetti again the second time. 
Maybe there's a way to stop the confetti, or maybe you'll have to create new canvases.

Reply:
yes .. i think i am creating cofetti again after some one wins or draw